# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wgpu/player/tests/data/quad.wgsl
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/gfx/wgpu/player/tests/data/quad.wgsl b/gfx/wgpu/player/tests/data/quad.wgsl
deleted file mode 100644
--- a/gfx/wgpu/player/tests/data/quad.wgsl
+++ /dev/null
@@ -1,16 +0,0 @@
-[[stage(vertex)]]
-fn vs_main([[builtin(vertex_index)]] vertex_index: u32) -> [[builtin(position)]] vec4<f32> {
-    // hacky way to draw a large triangle
-    let tmp1 = i32(vertex_index) / 2;
-    let tmp2 = i32(vertex_index) & 1;
-    let pos = vec2<f32>(
-        f32(tmp1) * 4.0 - 1.0,
-        f32(tmp2) * 4.0 - 1.0
-    );
-    return vec4<f32>(pos, 0.0, 1.0);
-}
-
-[[stage(fragment)]]
-fn fs_main() -> [[location(0)]] vec4<f32> {
-    return vec4<f32>(1.0, 1.0, 1.0, 1.0);
-}